### ISSUE TITLE
Enforce logit lens by pruning logits after each layer

### ISSUE THREAD

**Issue by quinn-dougherty** [EleutherAI/project-menu#2]
## Description

Enforce logit lens by pruning logits after each layer (needs elaboration)

## Motivation (optional)

_ 

## Point person (optional)

Gurkenglas 

## Deliverable (optional)

_ 

## Timeline (optional)

_
